colony/shell.py

```diff
@@ -101,7 +102,16 @@ def main():
 
     command_class = commands_table[input_parser.command]
     command = command_class(argv, conn)
-    command.execute()
+    result = command.execute()
+
+    _exit(result)

```

Comment:
why "_exit" method starts with underscore? its not a private method. Can you fix it?